db/migrate/20210817084817_validate_remove_null_from_participant_profile_schedules.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class ValidateRemoveNullFromParticipantProfileSchedules < ActiveRecord::Migration[6.1]
+  def change
+    validate_check_constraint :participant_profiles, name: "participant_profiles_schedule_id_null"
```

Comment:
I thought the whole point of strong_migrations was to have you do the former check_constraint, then run a required data change (in a console) and then a subsequent migration to run the validation and removal. I'd expect this to fail, force you to run a session to fix the data and then allow a redeploy and re-run.

Reply:
https://github.com/ankane/strong_migrations#setting-not-null-on-an-existing-column

Reply:
I did just what the docs said. It works locally. Presumably the comment in the docs, 
```
    # in Postgres 12+, you can then safely set NOT NULL on the column
```
is what is going wrong - our postgres is 11...